### PR TITLE
Updated support.md and fixed bugs in my code.

### PR DESCRIPTION
**Pull request by ssiddhantsharma**
Signed-off-by: Siddhant Sharma siddhaantsharma.ss@gmail.com

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -2,4 +2,8 @@
 
 Thanks for trying out Ark! We welcome all feedback, please consider joining our mailing list: 
 
-- [Mailing List](http://j.hept.io/ark-list)
\ No newline at end of file
+- [Mailing List](http://j.hept.io/ark-list)
+
+# Updates
+
+For the latest updates about us, follow [**@heptio**](https://twitter.com/heptio) on Twitter or visit the [**heptio blog!**](https://blog.heptio.com/)
```
[the remark this answers is not in the document]
Comment (ssiddhantsharma) [reply SUPPORT.md]: @ncdc Fixed the issue!

File: SUPPORT.md [hunk 2]
```diff
@@ -2,4 +2,8 @@
 
 Thanks for trying out Ark! We welcome all feedback, please consider joining our mailing list: 
 
-- [Mailing List](http://j.hept.io/ark-list)
\ No newline at end of file
+- [Mailing List](http://j.hept.io/ark-list)
+
+# Updates
+
+For the latest updates about us, follow [**@heptioark**](https://twitter.com/heptioark) on Twitter or visit the [**heptio blog!**](https://blog.heptio.com/)
```
Comment (ncdc) [inline SUPPORT.md]: Please change `@heptioark` to `@HeptioArk` (capital H, capital A). Please also change `**heptio blog**` to `**Heptio blog**` (capital H in Heptio).

Once you've made these changes, please squash your commits down to a single commit and we should be able to merge. Thanks!

Comment (ssiddhantsharma) [reply SUPPORT.md]: @ncdc I have made the required changes to the code.

Comment (ssiddhantsharma) [reply SUPPORT.md]: @skriss @ncdc I made the commits correctly this time, but I am not able to resolve the sign-off!
Can you close this without that?

Comment (ncdc) [reply SUPPORT.md]: @ssiddhantsharma we do need the signoff, and we'd like this to be squashed to a single commit. If you need assistance with either, please let us know.

**Comment by ssiddhantsharma** [comment]
@ncdc I am not able to commit the signed off by!
Should I create a new PR, as I am using the GitHub desktop and not command line, how can I create a sign-off?

**Comment by ncdc** [comment]
@ssiddhantsharma please don't create a new PR. We'll figure this out. I'm checking to see how we can do this.

**Comment by ncdc** [comment]
@ssiddhantsharma it doesn't look like GitHub Desktop supports this. Would you be willing to install the `git` command line client? That's the easiest way to resolve this.

**Comment by skriss** [comment]
@ssiddhantsharma do you want to try @ncdc's suggestion above in order to get this merged, or should we close this PR out?

**Comment by skriss** [comment]
closing - not ready for merge and no response from contributor.

**Pull request closed without merging**